apps/elf-council-frontend/scripts/scrape-delegates.sh

```diff
@@ -1,15 +1,21 @@
 # Mainnet only
+MAINNET_OUTPUT=src/elf-council-delegates/delegate_mainnet_scraped_results.json
+
+# Scrape the comments and create a list of json objects for each delegate comment
 curl -s "https://forum.element.fi/api/viewComments?chain=element-finance&root_id=discussion_4146" |
+  # collect the initial json objects for each delegate comment
   jq '{ 
   result: [.result[] 
     | select(.deleted_at = null)
-    | select(.plaintext | ascii_downcase | contains("statement of intent") or length > 280)
-    | select(.plaintext | ascii_downcase | contains("ethereum address"))
     | { 
 	commonwealthCommentId: .id,
-	"commonweathName": .Address["name"],
+	"commonweathName": (.Address["name"] // "Anonymous"),
```

Comment:
If the user has not set a name in commonwealth we get back null, but we should just use "Anonymous" as the label instead. `//` is jq speak for "or"